app/assets/javascripts/admin/enterprise_fees/controllers/enterprise_fees_controller.js.coffee

```diff
@@ -1,6 +1,6 @@
 angular.module('admin.enterpriseFees').controller 'enterpriseFeesCtrl', ($scope, $http, $window, enterprises, tax_categories, calculators) ->
   $scope.enterprises = enterprises
-  $scope.tax_categories =  [{id: -1, name: "Inherit From Product"}].concat tax_categories
+  $scope.tax_categories =  [{id: -1, name: ('js.admin.enterprise_fees.inherit_from_product' | t) }].concat tax_categories
```

Comment:
ah much better, now we can talk here :-)
the technique I use to fix these things is to look around in similar files, there are many points in the js code where things like this happen.
I would try this:
```suggestion
  $scope.tax_categories =  [{id: -1, name: "{{ 'js.admin.enterprise_fees.inherit_from_product' | t }}" }].concat tax_categories
```
and if it doesnt work, I'd use the t function:
```suggestion
  $scope.tax_categories =  [{id: -1, name: t('js.admin.enterprise_fees.inherit_from_product') }].concat tax_categories
```